### PR TITLE
feat(java install): move maven into collapsers

### PR DESCRIPTION
**Pull request by akristen**

### CODE REVIEW

File: src/install/java/download/maven.mdx [hunk 1]
```diff
@@ -10,55 +10,41 @@ This section explains how to configure Maven to download and unzip the `newrelic
...
+    With the New Relic zip file installed, Maven needs to unzip the file and access its contents. Go to `maven-dependency-plugin` in your `pom.xml`, then configure Maven so it can unzip `newrelic-java.zip`. 
 
-This will unzip all the files into the newly created `newrelic/` within `project.build.directory`:
+Below are two examples for configuring your `pom.xml` file:
 
 <Tabs>
-	<TabsBar>
+    <TabsBar>
 		<TabsBarItem id="zip-example">Zip example</TabsBarItem>
+        <TabsBarItem id="standalone">Snippet standalone"</TabsBarItem>
```
Comment (paperclypse) [inline src/install/java/download/maven.mdx:47]:
```suggestion
        <TabsBarItem id="standalone">Snippet standalone</TabsBarItem>
```

Deleting this weird extra quotation mark

**Review by paperclypse — DISMISSED** [review]
Looks good to me!

**Review by a-sassman — APPROVED** [review]

File: src/install/java/download/gradle.mdx [hunk 2]
```diff
@@ -10,34 +10,51 @@ When you use Gradle to download and unzip the `newrelic-java.zip` file, here's w
...
+  <Collapser
+    className="freq-link"
+    id="step1"
+    title="1. Configure `build.gradle`"
+  >
+Add the following snippet to your `build.gradle` file with the plugin you'll use to download the agent. For example:
+
+    ```java
     plugins {
         id "de.undercouch.download" version "3.4.3"
```
Comment (jeffalder) [inline src/install/java/download/gradle.mdx:23]: The latest version of this plugin is `5.3.0`, so someone should test this method with the new version.

Comment (akristen) [reply src/install/java/download/gradle.mdx:23]: Hi @jeffalder. Thanks for giving this update! Is there anyone I should reach out to to test this method with the most recent version?

Comment (akristen) [inline src/install/java/download/gradle.mdx:23]:
```suggestion
        id "de.undercouch.download" version "5.3.0"
```

**Review by paperclypse — APPROVED** [review]
Approved again!

**Pull request merged**